content/handbook/security/corporate/systems/google/calendar/delegation/_index.md

```
@@ -0,0 +1,62 @@
+
+1. カレンダー設定のドロップダウンメニューから `Share with specific people` を選択します。
+
+   <img src="/images/security/corporate/systems/google/calendar/delegation/google_calendar_settings.png" alt="Google Calendar Settings" width="200"/><br>
+
+1. `+ Add People` を選択します。
+
+1. カレンダーを共有する個人のメールアドレスを追加します。
+
+    <img src="/images/security/corporate/systems/google/calendar/delegation/google_calendar_settings_share.png" alt="Google Calendar Settings Share" width="400"/>
+
+1. `Permissions` ドロップダウンメニューを選択し、その個人に適切なアクセス権を付与します。[カレンダーの権限レベル](#calendar-permission-levels) を参照してください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)